src/extensions/extension.tsx

```diff
@@ -194,16 +200,27 @@ async function loadPageRecursive(type: RepoType, pageNum: number) {
   // The recursion isn't super clean...
 
   // TODO: re-enable this once everything works with mono-manifest...
-  // await Promise.all([
-  //   loadPageRecursive("extension", 1),
-  //   loadPageRecursive("theme", 1),
-  // ]);
+   await Promise.all([
```

Comment:
We should at least check for the setting here before sending off queries when we're not going to use them